dluhc-component-library/src/components/policyForm/components/MultiItem.tsx

```diff
@@ -0,0 +1,74 @@
+interface MultiItemProps {
+  label?: string;
+  values?: ReadonlyArray<string>;
+  className?: string;
+  onChange: (values: ReadonlyArray<string>) => void;
+}
+
+const MultiItem = ({
+  label,
+  values = [],
+  className,
+  onChange,
+}: MultiItemProps) => {
+  const handleValueChange = (index: number, newValue: string) => {
+    let newState = [...values];
+
+    newState[index] = newValue;
+
+    onChange(newState);
+  };
+
+  const handleAddClicked = () => {
+    onChange([...values, ""]);
+  };
+
+  const handleDeleteClicked = (index: number) => {
+    const newState = [...values];
+
+    newState.splice(index, 1);
+
+    onChange(newState);
+  };
+
+  const inputs = values.map((value, index) => {
+    return (
+      <div className="flex flex-row my-4">
```

Comment:
Could this div have a `key`, or would it have to be `index` if it did?

Reply:
Good spot, it should have a key. It annoyingly will have to be an index though.